Bottom: d687c2a7b298467031ac36a0a51054692cfcabad
Top:    a01515f2c40cbe5454eb7572a7146b81cc5bb05a
Author: Markus Hitter <mah@jump-ing.de>
Date:   2011-01-14 21:05:45 +0100

INSTALL: add instructions on how to create configure.

Closes-bug: lp-702484


---

diff --git a/INSTALL b/INSTALL
index c4df37d..62fc50e 100644
--- a/INSTALL
+++ b/INSTALL
@@ -2,6 +2,14 @@ PCB uses a standard GNU autoconf/automake based build
 system which should make compilation go smoothly on most unix-like
 systems.  Please do read this first section however.
 
+If there is no "configure" script, e.g. because you pulled sources
+from the Git repository, make sure you have autopoint and autoconf
+installed, then run:
+
+  ./autogen.sh
+
+This will set up and run autoconf to create configure.
+
 PCB is organized into a core program that deals with all of the
 internal database procedures and a collection of Human Interface
 Devices (HID's).  The HID's provide exporting/printing capability
